OsmAnd/res/layout/gpx_info_card.xml

```diff
@@ -51,8 +50,16 @@
 			android:id="@+id/location_container"
 			layout="@layout/item_with_title_desc" />
 
-	</LinearLayout>
+		<include
+			android:id="@+id/link_container"
+			layout="@layout/item_with_title_desc"
+			android:visibility="gone" />
 
-	<include layout="@layout/card_bottom_divider" />
+		<include
+			android:id="@+id/description_container"
+			layout="@layout/item_with_title_desc"
+			android:visibility="gone" />
```

Comment:
It is better to control visibility in code, also we can inflate necessary items in card